x-pack/test/functional/apps/maps/group1/auto_fit_to_bounds.js

```diff
@@ -30,7 +29,7 @@ export default function ({ getPageObjects, getService }) {
         expect(hits).to.equal('6');
 
         const { lat, lon } = await PageObjects.maps.getView();
-        expect(Math.round(lat)).to.equal(41);
+        expect(Math.round(lat)).to.be.within(41, 43);
```

Comment:
Can you add some context on how the test was failing and why this fixed it? Maybe some screen shots of the failed test? I would like to know why this value is not stable in tests.